SimG4CMS/Muon/interface/MuonSensitiveDetector.h

```diff
@@ -36,22 +29,18 @@ class UpdatablePSimHit;
 class MuonSubDetector;
 class MuonG4Numbering;
 class SimHitPrinter;
-class TrackInformation;
-class G4Track;
+class G4Step;
 class G4ProcessTypeEnumerator;
-class G4TrackToParticleID;
 class SimTrackManager;
 
 class MuonSensitiveDetector : 
 public SensitiveTkDetector,
-public Observer<const BeginOfEvent*>,
-public Observer<const EndOfEvent*>
```

Comment:
@civanch there is no action in the EndOfEvent, this is the reason to remove this line, am I correct?

Reply:
Yes, if methods will not be empty, they may be returned back.

Reply:
@fabiocos , one of the goals of this clean-up is to take out methods/code which doing nothing but is copy/paste from one SD class to another.